source/_components/climate.modbus.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+title: "Modbus"
+description: "Instructions how to integrate a Modbus thermostat within Home Assistant."
+date: 2018-01-29 9:35
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: modbus.png
+ha_category: Thermostat
+ha_release: pre 0.7
```

Comment:
This is a new component. Please set the `ha_release` to the next release (`0.66`).